### PR TITLE
Fixes tabling being shit

### LINKED ISSUE #14795: Tabling is shit
2 second stuns are lame, bring back 5 second stuns!

### PR DESCRIPTION
**Pull request by Iamgoofball**
Fixes #14795
:cl: MrGoofedOne
experimental: Fixes tabling being shit.
/:cl:

### CODE REVIEW

**Comment by lzimann** [comment]
top kek mate

**Comment by Ahammer18** [comment]
Doing god's work son

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -81,7 +81,7 @@
 /obj/structure/table/attack_hand(mob/living/user)
 	user.changeNext_move(CLICK_CD_MELEE)
 	if(tableclimber && tableclimber != user)
-		tableclimber.Weaken(2)
+		tableclimber.Weaken(5)
```
Comment (ChangelingRain) [inline code/game/objects/structures/tables_racks.dm]: goof fix your shit this isn't even the right thing

Comment (Iamgoofball) [inline code/game/objects/structures/tables_racks.dm]: WHOOPS

**Comment by Zilenan91** [comment]
Out. Tabling was cancer.

**Comment by lzimann** [comment]
Also, nice work around the feature freeze since you created an issue to "fix"

**Comment by Iamgoofball** [comment]
don't worry I'm just following sensei MrStonedOne's teachings 
https://github.com/tgstation/-tg-station/issues/14786
https://github.com/tgstation/-tg-station/pull/14787

**Comment by PKPenguin321** [comment]
posting in epic thread

:+1:-ing in epic thread

**Comment by xxalpha** [comment]
THIS DOESN'T COUNT FOR CHERIBUX

**Comment by Niknakflak** [comment]
This is cancer.  Fuck off goofball

**Comment by PKPenguin321** [comment]
rude

**Comment by duncathan** [comment]
:+1:

**Comment by PKPenguin321** [comment]
wew

**Comment by tkdrg** [comment]
Chain tabling was cancer please no

**Pull request closed without merging**